### DISCUSSION TITLE
IBM Professional Graphics Controller emulation in 86Box

### DISCUSSION THREAD

**Discussion by fire-lizard** [86Box/86Box#4678] [General]
Hi everyone.
While I had no problem emulating IBM 8514/a, which is sort of successor to the IBM PGC, I am having troubles with the PGC emulation in 86Box.
I have chosen "IBM PGC" as Display option, and IBM AT diagnostics disc shows that PGC is actually present in the system.
But despite the fact I am loading PGC drivers in config.sys, I could not get any software (like GKS or AutoCAD 2.6 or IBM PGC Demos) work and show me the nice PGC graphics, it just hangs the machine (AMI IBM AT 80286 with 512K RAM) and that's it.
Could anyone advise me about steps should be taken to emulate it properly?

**Comment by Torinde** [comment]
Related:
- #404
